torch/profiler/profiler.py

```diff
@@ -140,13 +140,11 @@ def __init__(
         if activities:
             self.activities = activities
         else:
+            self.activities = set([ProfilerActivity.CPU])
```

Comment:
better set default value for activities, that way it will also automatically show up in docs.

Reply:
since we're going to add cuda to the default list (when cuda is enabled) then I'll have to do the check in the code anyway

Reply:
Ok, I'd still set the default value to CPU, but up to you, then here you'd need to check only if cuda is available. 
Btw, what happens when user requests CUDA in activities, but cuda is not available? Is it ignored?

Reply:
we output an error in this case